src/test/java/org/ays/auth/controller/AysAuthControllerTest.java

```diff
@@ -238,7 +238,7 @@ void givenValidAysTokenInvalidateRequest_whenTokensInvalidated_thenReturnSuccess
     void givenValidForgotPasswordRequest_whenSendPasswordCreateMail_thenReturnSuccessResponse() throws Exception {
         // Given
         AysPasswordForgotRequest mockForgotPasswordRequest = new AysForgotPasswordRequestBuilder()
-                .withEmailAddress(AysValidTestData.User.EMAIL_ADDRESS)
+                .withEmailAddress(AysValidTestData.SuperAdmin.EMAIL_ADDRESS)
```

Comment:
Burayı neden değiştirdik?

Reply:
testler için değiştirildi

Reply:
Burada SuperAdmin yerine Admin yapabiliriz belki daha iyi olur

Reply:
Burada User çalıştı. ilk denediğimde çalışmamıştı başka bir sebepten olabilir. User olarak değiştireceğim

Reply:
Bu kullanıcının `institution:page` yetkisi yok diye biliyorum, bence bi tekrar kontrol edelim

Reply:
User olarak verdiğimde test success oluyor.

Reply:
Tamamdır tekrar kontrol edeceğim

Reply:
@sarpkan00 baktım, burayı değiştirmemiz bir şeyi değiştirmiyor çünkü unit test olarak davranıyor fakat `Admin` olması daha doğru o yüzden resolve ediyorum